config/database.yml

```diff
@@ -1,10 +1,16 @@
 development:
   adapter: postgresql
   database: growstuff_dev
+  host: localhost
+  user: growstuff
+  password: growstuff
 
 test:
   adapter: postgresql
   database: growstuff_test
+  host: localhost
+  user: growstuff
+  password: growstuff
```

Comment:
I've now fixed my local DB setup so I don't need to keep applying and stashing these changes :-)